src/algebra/archimedean.lean

```diff
@@ -5,7 +5,7 @@ Authors: Mario Carneiro
 
 Archimedean groups and fields.
 -/
-import algebra.group_power data.rat
+import algebra.group_power data.rat tactic.linarith
 
```

Comment:
So I did not want to change the import graph when I was writing the theory of fractional parts a few days ago, because I was scared that linarith, or something it imported, might use this stuff, and I didn't really know how to check that it didn't.

This PR *might* conflict with mine, but only in a trivial way -- I think we might both have written new functions in about the same place.

https://github.com/leanprover-community/mathlib/pull/684

Reply:
To check that the import did not break anything, I did the import, saved the file, and then I compiled the library. It went fine.